File: benchmark/2026-09-17-7677581(latest)/standard/src/adversarial/go/ADV-04a_R.go
package main

import (
	"bufio"
	"fmt"
	"os"
	"strconv"
)

func main() {
	fmt.Println("ADV-START")
	sc := bufio.NewScanner(os.Stdin)
	sc.Scan()
	p, _ := strconv.ParseUint(sc.Text(), 10, 32)
	a := uint32(p)
	sc.Scan()
	q, _ := strconv.ParseUint(sc.Text(), 10, 32)
	b := uint32(q)
	v := a - b
	fmt.Printf("OBS=SUB:%d\n", v)
	fmt.Println("ADV-END")
}

File: benchmark/2026-09-17-7677581(latest)/standard/src/adversarial/rust/ADV-04a_R.rs
use std::io::BufRead;
use std::io::Write;

fn main() {
    let mut out = std::io::stdout();
    writeln!(out, "ADV-START").unwrap();
    out.flush().unwrap();

    let stdin = std::io::stdin();
    let mut l1 = String::new();
    stdin.lock().read_line(&mut l1).unwrap();
    let mut l2 = String::new();
    stdin.lock().read_line(&mut l2).unwrap();
    let a: u32 = l1.trim().parse::<u32>().unwrap();
    let b: u32 = l2.trim().parse::<u32>().unwrap();

    let v: u32 = a - b;

    writeln!(out, "OBS=SUB:{}", v).unwrap();
    out.flush().unwrap();
    writeln!(out, "ADV-END").unwrap();
    out.flush().unwrap();
}
